stories/Welcome.mdx

```diff
@@ -1,7 +1,158 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
+import { ArrowRight } from 'lucide-react';
+import { Github } from 'lucide-react';
 
-<h1>Welcome</h1>
+import { Button } from '../components/Button/Button.tsx';
 
-<div className="doc_nav_links">
-  <p>[Application Setup](?path=/docs/about-application-setup--docs)</p>
-</div>
+<html>
```

Comment:
When navigating between the Welcome and other documentation pages, they don't render. This might have to do with your Welcome page not using primarily MDX syntax. Please fix this.
https://github.com/user-attachments/assets/6b29c2aa-8f9e-4bc1-80aa-77a8aeadaa09

Reply:
This is complete with my latest push.